packages/driver/src/cy/commands/sessions.ts

```diff
@@ -278,8 +278,12 @@ export default function (Commands, Cypress, cy) {
   }
 
   function throwIfNoSessionSupport () {
-    if (!Cypress.config('experimentalSessionSupport')) {
-      $errUtils.throwErrByPath('sessions.experimentNotEnabled')
+    if (!Cypress.config('experimentalLoginFlows')) {
```

Comment:
Should I fail the test if `experimentalSessionSupport` is enabled regardless of `experimentalLoginFlows`?

Reply:
No, it doesn't hurt anything if they're both enabled, besides extraneous config, but that doesn't' seem like a fail-able offense.